packages/sit-onyx/src/composables/asymmetricComputed.ts

```diff
@@ -0,0 +1,17 @@
+import { computed, type Ref, type WritableComputedOptions } from "vue";
+
+export type AsymmetricComputedOptions<GetterT, SetterT> = {
+  set: (v: SetterT) => void;
+  get: () => GetterT;
+};
+
+export interface AsymmetricComputedRef<GetterT, SetterT> extends Ref {
+  get value(): GetterT;
+  set value(v: SetterT);
+}
+
+/**
+ * Wrapper for `computed` that allows different typings for getter and setter.
+ */
+export const asymComputed = <GetT, SetT>(opts: AsymmetricComputedOptions<GetT, SetT>) =>
```

Comment:
I am not sure if this wrapper is confusing...since its only used once, can't we just use `as type XYZ` there and remove the `asymComputed`?

Reply:
I would prefer not to, as that provide worse types as there can be no inferred generics